consul-core/src/main/java/com/smoketurner/dropwizard/consul/core/ConsulAdvertiser.java

```diff
@@ -162,4 +173,18 @@ public void deregister() {
             LOGGER.error("Failed to deregister service from Consul", e);
         }
     }
+
+    private String buildHealthCheckUrl(int adminPort) {
+        StringBuilder healthCheckUrl = new StringBuilder("http://");
```

Comment:
Done.

2016-08-05 2:26 GMT+02:00 Justin Plock <notifications@github.com>:

> In consul-core/src/main/java/com/smoketurner/dropwizard/consul/
> core/ConsulAdvertiser.java
> <https://github.com/smoketurner/dropwizard-consul/pull/14#discussion_r73625707>
> :
>
> > @@ -162,4 +173,18 @@ public void deregister() {
> >              LOGGER.error("Failed to deregister service from Consul", e);
> >          }
> >      }
> > +
> > +    private String buildHealthCheckUrl(int adminPort) {
> > +        StringBuilder healthCheckUrl = new StringBuilder("http://");
>
> Instead of building a string here, is it possible to create a Uri
> instance then call its .toString() method instead?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/smoketurner/dropwizard-consul/pull/14/files/ff4852dfb1f94b1da966f533f42f138cc2a4fa84#r73625707>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAVUvqGSxtOAaVb5Y92dg2CoQkWkjnqmks5qcoM_gaJpZM4Jcxf9>
> .
>